packages/strapi-admin/admin/src/components/LeftMenuLinkContainer/index.js

```diff
@@ -43,7 +43,9 @@ function LeftMenuLinkContainer({ plugins, ...rest }) {
 
     return (
       <div key={j}>
-        <p className="title">{pluginsSections[current].name}</p>
+        <p className="title">
```

Comment:
Do you mind changing it to so we know that the DOM stays unchanged:

```
<p className="title">
  <FormattedMessage {...messaged.contentTypes}>
    {title => title}
  </FormattedMessage>
</p>
```

Reply:
Can you pls explain what exactly this does? Will change it of course :)

Reply:
`FormattedMessage` injects a `span` into the DOM when used as a Component. If you use it as a renderProps it returns the translated element. I ask you to use it as a render props so the DOM stays unchanged.